config/environments/production.rb

```diff
@@ -76,7 +76,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = true
+  config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
Did we end up needing this or was this left over from trying to figure out the UTF-8 parsing message?

Reply:
I changed that to fix a different deprecation issue after upgrading to Ruby 2.5. It's not related to the UTF-8 issue.